go/vt/vttest/vtprocess.go

```diff
@@ -247,6 +247,9 @@ func VtcomboProcess(env Environment, args *Config, mysql MySQLManager) *VtProces
 	if args.VSchemaDDLAuthorizedUsers != "" {
 		vt.ExtraArgs = append(vt.ExtraArgs, []string{"-vschema_ddl_authorized_users", args.VSchemaDDLAuthorizedUsers}...)
 	}
+	if *servenv.MySQLServerVersion != "" {
```

Comment:
This is cool. I thought we would have to add an arg.

Reply:
That was what I tried first, but it failed because this one was already being pulled in :)